Update for March 29, 2019 hot fix (#4084)
magento/devdocs: guides/v2.1/release-notes/ReleaseNotes2.1.17CE.md

## Apply patch Admin Dashboard Image-Charts to address deprecation of Google Image Charts

> Magento 2.x currently uses Google Image Charts to render static charts in Magento 2.x Admin dashboards. Google stopped supporting Google Image Charts on March 14, 2019, and Magento is providing the Admin Dashboard Image-Charts  patch to replace Google Image Charts with the Image-Charts free service. Users of Magento 2.x deployments will not be able to view static charts in Magento 2.x instances unless they download and apply this patch. 

Reviewer: ```suggestion
Magento 2.x currently uses Google Image Charts to render static charts in Admin dashboards. Google stopped supporting Google Image Charts on March 14, 2019, and Magento is providing the Admin Dashboard Image-Charts  patch to replace Google Image Charts with the Image-Charts free service. To re-enable static charts, you must download and apply this patch. 
```

Author: done

> This patch is now available for download. See  [Switch from deprecated Google Image Charts to Image-Charts for Magento](https://support.magento.com/hc/en-us/articles/360024850172) for information on downloading and applying this patch. 

Reviewer: ```suggestion
See [Switch from deprecated Google Image Charts to Image-Charts for Magento](https://support.magento.com/hc/en-us/articles/360024850172) for information on downloading and applying this patch. 
```
